OJS.Workers.ExecutionStrategies/CSharp/DotNetCore/DotNetCoreCompileExecuteAndCheckExecutionStrategy.cs

```diff
@@ -149,5 +167,32 @@ private void CreateRuntimeConfigJsonFile(string directory, string text)
 
             File.WriteAllText(jsonFilePath, text);
         }
+
+        private void AddUsingNamespacesForDotNetSix<T>(IExecutionContext<T> executionContext)
```

Comment:
This can be moved to `BaseExecutionStrategy` as a `virtual method` and be named something like `PreprocessCode` with default implementation that just returns the same code, i.e.:
```
virtual PreprocessCode(string code) {
    return code;
}
```

Then, this method can be called from `Execute` from `BaseExecutionStrategy`.

This will give us the infrastructure to add such _hooks_ for other strategies in the future.